extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -52,13 +52,11 @@ interface WebGLVideoFrameInfo {
 };
 
 [NoInterfaceObject]
-interface OES_EGL_image_external {
-    const GLenum TEXTURE_EXTERNAL_OES             = 0x8D65;
-    const GLenum SAMPLER_EXTERNAL_OES             = 0x8D66;
-    const GLenum TEXTURE_BINDING_EXTERNAL_OES     = 0x8D67;
-    const GLenum REQUIRED_TEXTURE_IMAGE_UNITS_OES = 0x8D68;
+interface WEBGL_video_texture {
+    const GLenum TEXTURE_VIDEO_IMAGE             = 0x851D;
+    const GLenum SAMPLER_VIDEO_IMAGE             = 0x8B61;
```

Comment:
Where did these enum values come from? I think that they need to be allocated out of a block that was given to the WebGL working group some time ago. This would imply that the names and values should be:

TEXTURE_VIDEO_IMAGE_WEBGL = 0x9248
SAMPLER_VIDEO_IMAGE_WEBGL = 0x9249

(The last enum allocated out of this range was MAX_CLIENT_WAIT_TIMEOUT_WEBGL in the WebGL 2.0 spec.)